tests/functional/z_cluster/nodes/test_nodes_restart.py

```diff
@@ -94,9 +95,9 @@ def test_nodes_restart(
             nodes.restart_nodes_by_stop_and_start(nodes=ocp_nodes, force=force)
 
         self.sanity_helpers.health_check()
-        self.sanity_helpers.create_resources(
-            pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory
-        )
+        retry(ServiceUnavailable, tries=20, delay=30)(
+            self.sanity_helpers.create_resources
+        )(pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory)
```

Comment:
Overall, the change looks good. But I have one concern: Do we see only the `ServiceUnavailable` exception? If there is a chance that there will also be a `CommandFailed` exception, I would also add it. 
I mean changing it to: 

> retry((ServiceUnavailable, CommandFailed), tries=20, delay=30)(
            self.sanity_helpers.create_resources
        )(pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory)

Reply:
As of now i have observed only service unavailable exception, as post disruption it takes some time for the cluster to except new requests. will add other exceptions if observed in later stage

Reply:
Okay, np.